fabmukunzi/certificate-manager---Fabrice
# Task3 KAN-49: Create new certificate

#### PR description

- Adds create certificate functionality

#### Ticket / Jira Link

- [KAN-49](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?selectedIssue=KAN-49)

#### Checklist:


#### How To Test Changes

- Checkout to this branch `feature/task3-KAN-49-create-new-certificate`
- Type `npm i` to install packages
- Open the terminal and type `npm run dev` to run the app
- Open your browser and navigate to `http://localhost:3000/`
- On the sidebar, click on example 1 and you should see a a button for "New certificate" Click on it you will see a form to create a new certificate

#### Screenshots:
![image](https://github.com/user-attachments/assets/3225abfa-a3ce-4d0e-9b9b-32e6b1626130)
